test/widgets/lightbox_test.dart

```diff
@@ -447,6 +448,35 @@ void main() {
       debugNetworkImageHttpClientProvider = null;
     });
 
+    testWidgets('download button triggers download', (tester) async {
+      prepareBoringImageHttpClient();
+      final message = eg.streamMessage(sender: eg.otherUser);
+      await setupPage(tester, message: message, thumbnailUrl: null);
+
+      final downloadButton = find.byIcon(Icons.download);
+
+      if (Platform.isAndroid){
+        expect(downloadButton, findsOneWidget);
```

Comment:
We use `check` instead of `expect`:

```suggestion
        check(findDownloadButton).findsOne();
```

(see other tests for examples)